lib/allegro/webapi/client.rb

```diff
@@ -1,7 +1,10 @@
+require 'timeout'
+
 module Allegro
   module WebApi
     class Client
       extend Forwardable
+      DELAY = 10
```

Comment:
`TIMEOUT`. Also, please allow that to be changed. I don't like the fact this is hardcoded and can't be changed easily.